sql/core/src/test/scala/org/apache/spark/sql/IntegratedUDFTestUtils.scala

```diff
@@ -749,6 +749,363 @@ object IntegratedUDFTestUtils extends SQLHelper {
     val prettyName: String = "Python UDTF whose 'analyze' method sets state and reads it later"
   }
 
+  object TestPythonUDTFInvalidEvalReturnsNoneToNonNullableColumnScalarType extends TestUDTF {
+    val name: String = "TestPythonUDTFInvalidEvalReturnsNoneToNonNullableColumnScalarType"
```

Comment:
It would be great if we could make this name a bit shorter :)

Reply:
Sounds good, I made all these invalid test UDTF names shorter!